client/lib/ffi.rs

```diff
@@ -878,3 +912,98 @@ impl TryInto<super::SessionStrParams<'static>> for casper_session_params_t {
         })
     }
 }
+
+/// Container for `DictionaryItemStrParams` construction options.
+///
+/// See [DictionaryItemStrParams](super::DictionaryItemStrParams) for more info.
+#[allow(non_snake_case, missing_docs)]
```

Comment:
Rust seems to be content even without `non_snake_case` 🐍 
Can you double-check?

Reply:
Well spotted.  Looks like `non_snake_case` is no longer required for types which are `#[repr(C)]`.  Please remove all the `allow(non_snake_case)` attributes in this file.

Reply:
Removed